packages/ibm-products-web-components/package.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "@carbon/ibm-products-web-components",
+  "description": "Carbon for IBM Products Web Components",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "main": "es/index.js",
+  "module": "es/index.js",
+  "type": "module",
+  "bugs": "https://github.com/carbon-design-system/ibm-products/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/carbon-design-system/ibm-products.git",
+    "directory": "packages/ibm-products-web-components"
+  },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
+  },
+  "keywords": [
+    "carbon",
+    "carbon design system",
+    "carbon community",
+    "carbon for ibm products",
+    "carbon for ibm products web components",
+    "web components"
+  ],
+  "scripts": {
+    "dev": "vite",
+    "build": "yarn clean && node tasks/build.js",
+    "build:storybook": "storybook build",
+    "clean": "rimraf es lib scss dist storybook-static",
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 3000",
+    "test": "vitest run",
+    "test:ui": "vitest --ui"
```

```suggestion
    "build": "yarn clean && node tasks/build.js && yarn wca",
    "build:storybook": "storybook build",
    "clean": "rimraf es lib scss dist storybook-static",
    "preview": "vite preview",
    "storybook": "storybook dev -p 3000",
    "test": "vitest run",
    "test:ui": "vitest --ui",
    "wca": "web-component-analyzer analyze src --outFile custom-elements.json"
```

Can add `wca` to the build command to generate `custom-elements.json` on each build.